fastchat/model/model_adapter.py

```diff
@@ -49,6 +49,13 @@
 )
 
 
+CLAUDE_NAMES = (
```

Comment:
Could you change `CLAUDE_NAMES` to `ANTHROPIC_MODEL_LIST`?

Reply:
Yeah, ANTHROPIC_MODEL_LIST seems better